x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/find_cases.ts

```diff
@@ -394,7 +394,7 @@ export default ({ getService }: FtrProviderContext): void => {
           expect(cases.cases[0].title).to.equal(uuid);
         });
 
-        it('should successfully find a case with a valid uuid in title', async () => {
```

Comment:
why were these two tests with the same name not throwing a listing error!?

Reply:
no idea why 😆